inc/class-cachify.php

```diff
@@ -807,15 +774,22 @@ public static function flush_notice() {
 	 * @since   0.1.0
 	 * @change  2.1.2
 	 *
-	 * @param   integer $id  Comment ID.
+	 * @param   integer	$id			Comment ID
+	 * @param   array	$comment	Comment data
 	 */
-	public static function edit_comment( $id ) {
-		if ( self::$options['reset_on_comment'] ) {
-			self::flush_total_cache();
-		} else {
-			self::remove_page_cache_by_post_id(
-				get_comment( $id )->comment_post_ID
-			);
+	public static function comment_edit( $id, $comment ) {
```

Comment:
I wouldn't rename this method, as it's a breaking change.
Making the second parameter optional would also be nice, to not break the current API.

Reply:
only 1.7% of all wp users are left on wp 4.4 and 4.5
if i am right automatic plugin updates were only introduced since wp 5.5
but it's up to you

Reply:
It’s not only about WP, but about the code. Wouldn’t care too much about WP <4.6. But we have no idea who might be using this method, potentially called from any other plugin or some custom hook for whatever reason that might be.

Reply:
In general. Sorry, but it's only I don't know. I don't want to blame you or feel blamed. How would you like to improve something without breaking or to modify **internal** things from time to time. Especially when it comes to the performance of a caching plugin.
If it's about code readability and you make everything twice - there will be no benefit for new coders. So there should be a renaming where it is needed for **one release** and leave it as is.

I edited it because it was the same function name as the WP function. There could be some confusion for new coders who want to read the code.

Reply:
I think @stklcode has a point, we should avoid renaming this method.

Reply:
From my point of view: internal === private

If there are good reasons to change things, we should change and raise the version accordingly. You, @timse201 are absolutely right, changes need changes and from time to time they are breaking and that’s fine.

Your reasons are good, so we might rename it, but to not mess up with the community, we should either
* keep and deprecate the old method or
* announce the change before the release

An unused, deprecated method does not do any harm here and gives other users/devs the chance to migrate before it gets removed.

Reply:
Let's take the deprecation route and keep the old method around for now instead of simply renaming it.

Reply:
resolved in feab290d58cb32083c8fd2ae105930cab2dac7d3